### PR TITLE
Implemented basic admin interface

### PR DESCRIPTION
**Pull request by wachterjohannes**
| Q | A
| --- | ---
| Bug fix? | no
| New feature? | yes
| BC breaks? | no
| Deprecations? | no
| Fixed tickets | none
| Related issues/PRs | none
| License | MIT
| Documentation PR | none

#### What's in this PR?

This PR implements a basic admin-interface.

### CODE REVIEW

**Comment by nickvanderzwet** [comment]
What's is the status of this PR, seems very useful :)

**Comment by wachterjohannes** [comment]
@nickvanderzwet we will force that in the next 3 weeks (:

**Comment by alexander-schranz** [comment]
add sorting to the datagrid: `url: Manager.url() + '?sortBy=created&sortOrder=desc',`

**Comment by alexander-schranz** [comment]
add tab `comments` to threads

File: Controller/CommentController.php [hunk 1]
```diff
@@ -0,0 +1,206 @@
...
+     * @Post("/comments/{id}")
+     *
+     * @param int $id
+     * @param Request $request
+     *
+     * @return Response
+     *
+     * @throws RestException
+     */
+    public function postTriggerAction($id, Request $request)
```
Comment (alexander-schranz) [inline Controller/CommentController.php]: its the only way to create comments so this is the postAction

**Comment by alexander-schranz** [comment]
toggler without any description

bildschirmfoto 2016-09-28 um 11 46 55

**Pull request merged**